aptos/contracts/sources/price_feed.move

```diff
@@ -0,0 +1,54 @@
+module pyth::price_feed {
+    use pyth::price_identifier::PriceIdentifier;
+    use pyth::price_status::PriceStatus;
+    use pyth::price::Price;
+
+    /// PriceFeed represents a current aggregate price for a particular product.
+    struct PriceFeed has copy, drop, store {
+        /// The price identifier
+        price_identifier: PriceIdentifier,
+        /// The status of the current aggregate price
+        status: PriceStatus,
+        /// The current aggregate price
+        price: Price,
+        /// The current exponentially moving average aggregate price
+        ema_price: Price,
+        /// The most recent previous price with TRADING status
+        previous_price: Price,
```

Comment:
should we drop this and make `price` the most recent price with TRADING status (per @ali-bahjati 's changes to the rust / solidity sdk?)